From 4066425505507fb3caf091ccae8e417997583a62 Mon Sep 17 00:00:00 2001
From: Caedmon Judd <caedmon@statebuilt.com>
Date: Thu, 20 Oct 2011 14:03:54 -0500
Subject: [PATCH 08/55] Notes work

---
 app/controllers/notes_controller.rb |   11 +++++++++--
 1 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/app/controllers/notes_controller.rb b/app/controllers/notes_controller.rb
index 57953c7..d2981cd 100644
--- a/app/controllers/notes_controller.rb
+++ b/app/controllers/notes_controller.rb
@@ -48,9 +48,16 @@ class NotesController < ApplicationController
   
   def delete
     @note = Note.find(params[:id], current_user)
-    @note.destroy
     
-    redirect_to '/'
+    if @note.destroy
+      redirect_to '/'
+    else
+      flash[:alert] = []
+      @note.errors.each do |error|       
+        flash.now[:alert] << error
+      end
+      render action: "index"
+    end
   end
   
 end
-- 
1.7.0.4

