From 0b81c251cbbb95aaaff4efaaa724fe6cfae6be46 Mon Sep 17 00:00:00 2001
From: Caedmon Judd <caedmon@statebuilt.com>
Date: Fri, 21 Oct 2011 14:46:18 -0500
Subject: [PATCH 14/55] Added current_admin

---
 app/controllers/application_controller.rb |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/app/controllers/application_controller.rb b/app/controllers/application_controller.rb
index c0747a2..d36ab5b 100644
--- a/app/controllers/application_controller.rb
+++ b/app/controllers/application_controller.rb
@@ -9,6 +9,9 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def current_auth
+    Hashie::Mash.new( {'id' => cookies.signed[:a], 'username' => cookies.signed[:b], 'password' => cookies.signed[:c]} ) if cookies.signed[:a]  
+  end
   def current_user
     unless @current_user
       @current_user = User.find(cookies.signed[:a]) if cookies.signed[:a]
-- 
1.7.0.4

