From 0f2c7de3a3ba68b261b30a4075ce1bb6f3f39dc5 Mon Sep 17 00:00:00 2001
From: Caedmon Judd <caedmon@statebuilt.com>
Date: Thu, 27 Oct 2011 07:46:33 -0500
Subject: [PATCH 34/55] Updates for Auth Branch pull request

---
 app/models/note.rb |    7 +------
 1 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/app/models/note.rb b/app/models/note.rb
index 188cb72..b32acdd 100644
--- a/app/models/note.rb
+++ b/app/models/note.rb
@@ -143,12 +143,7 @@ class Note
   def self.build_objects(response, auth)
     @return_notes = []
     response.each do |note|
-      @note = Note.new(note)
-      @note.auth = auth
-      #@note.content = @note.content_html
-      @note.content = @note.content_text
-      @note.reference = hash_to_osis(@note.reference)
-      @return_notes << @note      
+      @return_notes << build_object(note, auth)
     end
     @return_notes
   end
-- 
1.7.0.4

