From 036100052ffce1d12083c4d554ce349592c744f3 Mon Sep 17 00:00:00 2001
From: Caedmon Judd <caedmon@statebuilt.com>
Date: Fri, 28 Oct 2011 09:16:20 -0500
Subject: [PATCH 39/55] Minor updates

---
 app/models/note.rb |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/models/note.rb b/app/models/note.rb
index 5d1ae6f..125c2ab 100644
--- a/app/models/note.rb
+++ b/app/models/note.rb
@@ -65,7 +65,7 @@ class Note
     
   def create
     @token = Digest::MD5.hexdigest "#{auth.username}.Yv6-#{auth.password}"
-    @content = '<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE yv-note SYSTEM "http://' << Cfg.api_root << '/pub/yvml_1_0.dtd"><yv-note>' << @content << '</yv-note>'
+    @content = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE yv-note SYSTEM \"http://#{Cfg.api_root}/pub/yvml_1_0.dtd\"><yv-note>#{@content}</yv-note>"
     @reference = @reference.gsub('+', '%2b')
 
     response = YvApi.post('notes/create', class_attributes(:title, :content, :language_iso, :reference, :version,
@@ -80,7 +80,7 @@ class Note
   def update(id, fields)
     set_class_values(self, fields)
     @token = Digest::MD5.hexdigest "#{auth.username}.Yv6-#{auth.password}"
-    @content = '<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE yv-note SYSTEM "http://' << Cfg.api_root << '/pub/yvml_1_0.dtd"><yv-note>' << @content << '</yv-note>'
+    @content = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE yv-note SYSTEM \"http://#{Cfg.api_root}/pub/yvml_1_0.dtd\"><yv-note>#{@content}</yv-note>"
     @reference = @reference.gsub('+', '%2b')
 
     response = YvApi.post('notes/update', class_attributes(:id, :title, :content, :language_iso, :reference, :version,
-- 
1.7.0.4

