From 032c41ad9a03387f625faa8d54770e9569b40f58 Mon Sep 17 00:00:00 2001
From: Caedmon Judd <caedmon@statebuilt.com>
Date: Mon, 31 Oct 2011 14:01:51 -0500
Subject: [PATCH 44/55] Notes work

---
 app/controllers/notes_controller.rb |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/app/controllers/notes_controller.rb b/app/controllers/notes_controller.rb
index a2a795e..6ab93ba 100644
--- a/app/controllers/notes_controller.rb
+++ b/app/controllers/notes_controller.rb
@@ -25,12 +25,6 @@ class NotesController < ApplicationController
     if current_auth
       @note = Note.find(params[:id], current_auth)
       @note.reference = @note.reference.osis_noversion
-<<<<<<< HEAD
-      @note.version = @note.version.osis
-      @url_path = notes_path << '/' << params[:id]
-      @url_method = :put
-=======
->>>>>>> 5d49944310610a8f0c61784cf97925f670c3cff0
     else
       redirect_to notes_path
     end    
-- 
1.7.0.4

